third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -2160,6 +2190,26 @@ resource "google_compute_instance_template" "foobar" {
 `, suffix)
 }
 
+func TestAccComputeInstanceTemplate_validDiskType(t *testing.T) {
```

Comment:
We should group this right next to TestAccComputeInstanceTemplate_invalidDiskType.

Also, `_withScratchDisk` maybe?